File: WebApp/Scripts/app/model/MI_CONSUME_DATE.js
Ext.define('WEBAPP.model.MI_CONSUME_DATE', {
    extend: 'Ext.data.Model',
    fields: [
        { name: 'DATA_DATE', type: 'string' },
        { name: 'DATA_BTIME', type: 'string' },
        { name: 'DATA_ETIME', type: 'string' },
        { name: 'WH_NO', type: 'string' },
        { name: 'MMCODE', type: 'string' },
        { name: 'VISIT_KIND', type: 'string' },
        { name: 'CONSUME_QTY', type: 'string' },
        { name: 'STOCK_UNIT', type: 'string' },
        { name: 'INSU_QTY', type: 'string' },
        { name: 'HOSP_QTY', type: 'string' },
        { name: 'PARENT_ORDERCODE', type: 'string' },
        { name: 'PARENT_CONSUME_QTY', type: 'string' },
        { name: 'CREATEDATETIME', type: 'string' },
        { name: 'PROC_ID', type: 'string' },
        { name: 'PROC_MSG', type: 'string' },
        { name: 'PROC_TYPE', type: 'string' }
    ]
});

File: WebApp/Models/MI/MI_CONSUME_DATE.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApp.Models
{
    public class MI_CONSUME_DATE : JCLib.Mvc.BaseModel
    {
        public string DATA_DATE { get; set; }
        public string DATA_BTIME { get; set; }
        public string DATA_ETIME { get; set; }
        public string WH_NO { get; set; }
        public string MMCODE { get; set; }
        public string VISIT_KIND { get; set; }
        public string CONSUME_QTY { get; set; }
        public string STOCK_UNIT { get; set; }
        public string INSU_QTY { get; set; }
        public string HOSP_QTY { get; set; }
        public string PARENT_ORDERCODE { get; set; }
        public string PARENT_CONSUME_QTY { get; set; }
        public string CREATEDATETIME { get; set; }
        public string PROC_ID { get; set; }
        public string PROC_MSG { get; set; }
        public string PROC_TYPE { get; set; }


        public string WH_NAME { get; set; }
        public string MMNAME_E { get; set; }


    }
}